### DISCUSSION TITLE
Add `providerType` in returned session with 'database' strategy?

### DISCUSSION THREAD

**Discussion by baptisteArno** [nextauthjs/next-auth#3578] [Help] [answered]
I'm trying to figure out how I can add `account.type` into the returned `session` when using a database.

I managed to do it with the `jwt` strat (in the `jwt` callback) but what about with `database`?

**Comment by balazsorban44** [comment, marked as answer]
If you use a single provider, it might be unnecessary, you can just assume the same value all the time.

Considering this question, it's probably not the case.

When using a DB persisted session, we don't fetch the account info by default, you will have to retrieve it yourself, possibly in the `session` callback.
